test/test_services_constructor.jl

```diff
@@ -172,3 +172,42 @@ end
     # These values might change as the AGC model is refined
     moi_tests(agc_problem, false, 720, 0, 480, 0, 384, false)
 end
+
+@testset "Test GroupReserve from Thermal Dispatch" begin
```

Comment:
Not sure if this is the intended way for these tests, please let me know and I will correct it.

Reply:
can you explain the tests to me?
Am i right in saying that all the code here is setup, then all the actual tests are in [`moi_tests`](https://github.com/NREL-SIIP/PowerSimulations.jl/blob/3816aae989a3c4da286b85fd79e081f7bb369f02/test/test_utils/model_checks.jl#L4)?
i'm not yet familiar with the tests in this package. AFAICT this matches the tests in this file. 👍  just wonder if there are any additional tests, e.g. unit tests for some of the functions added here, that can/should also be added?

Reply:
MOI tests are meant to do the testing that makes sure the correct number of constraints and variables exist in the container. It also checks that the model uses parameters adequately.